src/mmw/sass/base/_map.scss

```diff
@@ -24,6 +24,15 @@
 .leaflet-draw-toolbar a {
 	background-image: url('../images/spritesheet.png');
 }
+
 .leaflet-retina .leaflet-draw-toolbar a {
 	background-image: url('../images/spritesheet-2x.png');
 }
+
+.leaflet-control-layers-toggle{
+  background-image: url('/static/images/layers.png');
```

Comment:
I don't think we should reference `static` in the path, because it can change. FYI, it is configured here: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/mmw/settings/base.py#L177.

Instead, we should use a relative path.